src/components/Auth.js

```diff
@@ -66,6 +82,35 @@ class Auth extends Component {
     return false
   }
 
+  async onCreate (e) {
+    e.preventDefault()
+
+    // setting the model dialog state to be true 
+    this.setState({
+      "modelOpen": true
+    })
+
+    
+  }
+
+  async onSignUp (e) {
+    e.preventDefault()
+
+    const { data: { addSelfUser } } = await this.props.SignUp({
```

Comment:
We can change the mutation name to camel-case, as the other mutations from the project: signUp